.github/workflows/build-and-test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        jdk: [8, 11]
+        jdk: [8.0.192, 8, 11.0.3, 11]
```

Comment:
What's the purpose of having both major and patch version matrix versions?

Reply:
@tomakehurst,
 
Glad you asked.

In short, it's good practice because vendors or projects that use this library often are in production with major fixed release versions, and the latest build can/has failed tests due to something in the build and nothing to do with your code.

Here's my write-up about fixed (major) releases:
[https://foojay.io/today/github-actions-with-java-part-2/](url)

But if you prefer I can remove the fixed versions. It's up to you.
Thank you,
Carl